src/twisted/_version.py

```diff
@@ -1,11 +1,7 @@
-# Copyright (c) Twisted Matrix Laboratories.
-# See LICENSE for details.
+# This file is auto-generated! Do not edit!
+# Use `python -m incremental.update Twisted` to change this file.
 
-# This is an auto-generated file. Do not edit it.
+from incremental import Version
 
-"""
-Provides Twisted version information.
-"""
-
```

Comment:
For some reason when I was looking at this I must have thought that was removing this docstring from `twisted.python.version` (which is just a compatibility alias).  _This_ docstring should probably stick around.